examples/magento-graphcms/pages/p/[url].tsx

```diff
@@ -78,6 +79,8 @@ function ProductConfigurable(props: Props) {
       />
       <ProductPageMeta {...product} />
 
+      <ProductPageBreadcrumb {...product} />
```

Comment:
This should be placed somewhere on the page where it isn't occluded by the `< Back` button, we can probably discuss where this should go, even if we don't included it here in the final PR.